pkgs/by-name/gp/gpustat/package.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, fontconfig
+, libGL
+, libX11
+, libXcursor
+, libXi
+, libXrandr
+, cmake
+, libxkbcommon
+, wayland
+, makeWrapper
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "gpustat";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "arduano";
+    repo = "gpustat";
+    rev = "v${version}";
+    sha256 = "sha256-zJuadQfyGJemjvkigC77Ql112w5alFrFg3hRnX9io58=";
+  };
+
+  cargoSha256 = "sha256-GNYEc7XYmO1tSqJx0KVPBW/Wq8PmjDpJBle0NYj7bkI=";
+
+  nativeBuildInputs = [ pkg-config cmake makeWrapper ];
+  buildInputs = [
+    fontconfig
+    libGL
+    libxkbcommon
+    libX11
+    libXcursor
+    libXi
+    libXrandr
+    wayland
+  ];
+
+  postInstall = ''
+    mkdir -p $out/share/applications
+    mkdir -p $out/share/pixmaps
+
+    cp $src/assets/gpustat.desktop $out/share/applications
+    cp $src/assets/icon_* $out/share/pixmaps
+  '';
+
+  # Wrap the program in a script that sets the LD_LIBRARY_PATH environment variable
+  # so that it can find the shared libraries it depends on.
+  # https://github.com/emilk/egui/issues/2486
```

Comment:
The comment is superflous - and misleading, since it cites egui, an unrelated project.

```suggestion
```

Reply:
My thought was that it's generally weird to have to wrap a Rust program in nix, but with egui programs there's an open issue that it needs to have those dependencies in LD_LIBRARY_PATH to work correctly.

This project uses egui btw.

Should I reword it somehow to make that even more clear, or should I just remove it still?

Reply:
Hum, egui is a vendored dependency?
If yes then you should make this fact a bit more clear.

Reply:
Should be updated